mirar/pipelines/winter/load_winter_image.py

```diff
@@ -210,12 +210,24 @@ def load_winter_stack(
     data, header = open_fits(path)
 
     dirname = path.split("/winter/")[0] + "/winter/"
+    logger.debug(dirname)
```

Comment:
Intended to be here? If so, please make the message meaningful

Reply:
removed